tough-kms/Cargo.toml

```diff
@@ -16,16 +16,15 @@ aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-kms/rustls"]
 [dependencies]
 tough = { version = "0.16", path = "../tough", features = ["http"] }
 ring = { version = "0.17", features = ["std"] }
-aws-sdk-kms = "0.36"
-aws-config = "0.57"
+aws-sdk-kms = { version = "1.4", features = ["behavior-version-latest"] }
+aws-config = { version = "1.0", features = ["behavior-version-latest"] }
 snafu = { version = "0.7", features = ["backtraces-impl-backtrace-crate"] }
 tokio = { version = "1", features = ["fs", "io-util", "time", "macros", "rt-multi-thread"] }
 pem = "3"
 
 [dev-dependencies]
-aws-smithy-runtime = {version = "0.57", features = ["test-util"]}
-aws-smithy-http = "0.57"
-aws-smithy-types = "0.57"
+aws-smithy-runtime = {version = "1.0", features = ["test-util"]}
+aws-smithy-types = "1.1"
```

Comment:
Should we make types "1.0" too and make sure `aws-smithy-runtime-api` and `aws-smithy-async` are also 1.0.3? (not sure what happens if we have mixed versions)

Reply:
We should make these say `"1"`, not `"1.1"`.